components/checkbox/stories/checkbox.stories.js

```diff
@@ -58,7 +68,16 @@ export default {
 			type: { name: "boolean" },
 			table: {
 			type: { summary: "boolean" },
-			category: "Component",
+			category: "State",
+			},
+			control: "boolean",
+		},
+		isReadOnly: {
```

Comment:
Is the expected behavior of this just that the checkboxes aren't interactive anymore?